toon.py

```diff
@@ -12,31 +12,49 @@
 import homeassistant.helpers.config_validation as cv
 
 # Home Assistant depends on 3rd party packages for API specific code.
-REQUIREMENTS = ['toonlib==0.4.0']
+REQUIREMENTS = ['toonlib==0.5.0']
 
 _LOGGER = logging.getLogger(__name__)
 
 DOMAIN = 'toon'
 TOON_HANDLE = 'toon_handle'
+CONF_GAS = 'gas'
+DEFAULT_GAS = True
+CONF_SOLAR = 'solar'
+DEFAULT_SOLAR = False
```

Comment:
I don't know, all the stuff we're doing interacting with the api takes a long time as it is. I regularly see messages like The initiation of sensor x took more than 10 seconds. I don't think waiting for a complete response will help us, it will just take like forever for the component to load. We should probably differentiate between items that are in the json response but are of zero value and items that are not in the json at all. I know you don't like it, but in those last cases, we can only make a decision if we rely on user input. Because of the flakiness of the api, we may need to have optional config items for each of the sensors and switches that let users force-turn them on or off